pkgs/servers/sql/cockroachdb/deps.nix

```diff
@@ -95,17 +95,17 @@
     fetch = {
       type = "git";
       url = "https://github.com/cockroachdb/cockroach";
```

Comment:
This could be removed as it would be fetched from `default.nix` already. `go2nix` before **1.1.1** didn't check this and cloud provide duplicates in `default.nix` and `deps.nix` like this one.